### PR TITLE
Add Kirby Return to Dreamland Metafortress patch

### PR DESCRIPTION
**Pull request by gamemasterplc**
Only added for US and JP version. Based off gameconfig patch included in USB loaders. PAL version gameconfig patch exists but has not been ported to Dolphin format. This is required to run gecko codes at all. Both versions have glitches in extras mode.

### CODE REVIEW

**Comment by spycrab** [comment]
@dolphin-emu-bot rebuild

File: Data/Sys/GameSettings/SUKJ01.ini [hunk 1]
```diff
@@ -0,0 +1,1395 @@
...
+0x80704F9C:dword:0x60000000
+0x80705C64:dword:0x48000088
+0x80705EE8:dword:0x4800008C
+0x8070605C:dword:0x60000000
+0x80706308:dword:0x60000000
+0x80706668:dword:0x60000000
+0x80706C94:dword:0x48000090
+0x80706E58:dword:0x48000090
+0x8070705C:dword:0x48000094
+0x80670F60:dword:0x60000000
```
Comment (spycrab) [inline Data/Sys/GameSettings/SUKJ01.ini]: I think there should be a newline here.

File: Data/Sys/GameSettings/SUKE01.ini [hunk 2]
```diff
@@ -0,0 +1,1396 @@
...
+0x80706C68:dword:0x60000000
+0x80707930:dword:0x48000088
+0x80707BB4:dword:0x4800008C
+0x80707D28:dword:0x60000000
+0x80707FD4:dword:0x60000000
+0x80708334:dword:0x60000000
+0x80708960:dword:0x48000090
+0x80708B24:dword:0x48000090
+0x80708D28:dword:0x48000094
+0x80672C20:dword:0x60000000
```
Comment (spycrab) [inline Data/Sys/GameSettings/SUKE01.ini]: Ditto.

**Comment by MayImilae** [comment]
But... I have this game and metafortress isn't causing Dolphin any trouble here. Why include the patch at all?

**Comment by JMC47** [comment]
The patch is required because if you want to use other cheats, you need to patch the game first.

**Comment by MayImilae** [comment]
Ahhh.

**Comment by leoetlino** [comment]
@dolphin-emu-bot allowmerge

LGTM after adding the missing trailing newlines in the INI files.

**Comment by gamemasterplc** [comment]
Depending on text editor, the last newline might not be visible however

**Comment by RisingFog** [comment]
@dolphin-emu-bot rebuild

**Pull request merged**